compiler/src/iree/compiler/Dialect/HAL/Target/VulkanSPIRV/VulkanSPIRVTarget.cpp

```diff
@@ -182,14 +182,43 @@ class VulkanSPIRVTargetBackend : public TargetBackend {
       return variantOp.emitError() << "should contain some spirv.module ops";
     }
 
+    DenseMap<StringRef, uint64_t> entryPointOrdinals;
+    uint64_t ordinalCount = 0;
```

Comment:
Do you ever see this map having a different size than `ordinalCount`?  It's not clear to me if we can have gaps in ordinals or not.

Reply:
If no, can we use the map size? If yes, should we call it something like 'maxOrdinal`?

Reply:
Previous passes in IREE's SPIR-V CodeGen guarantees that we have no gaps in ordinals. So it's a bit moot point from that perspective. I'm fine both ways. Changed.